src/mpi_serial/CMakeLists.txt

```diff
@@ -86,9 +86,14 @@ ecbuild_add_library( TARGET mpi_serial
 
 ## Install and Export ##
 
-install(
-    FILES
-        ${CMAKE_CURRENT_SOURCE_DIR}/mpif.h
-    DESTINATION
-        include/mpi_serial
-)
+if( ${HAVE_DUMMY_MPI_HEADER} )
+
+  install(
+      FILES
+          ${CMAKE_CURRENT_SOURCE_DIR}/mpif.h
+      DESTINATION
+          include/mpi_serial
+  )
+
+endif()
```

Comment:
Was this a problem?
Normally this would be installed in "include/mpi_serial/mpi.h" and not be part of any regular include dir for downstream packages unless these packages link with mpi_serial cmake target.

Reply:
@RyadElKhatibMF can you answer Willem ?

Reply:
@RyadElKhatibMF  thank you for the explanation!